packages/express-did-auth/src/factories/logout-factory.ts

```diff
@@ -0,0 +1,10 @@
+import { SessionManager } from '../types'
+
+export default function logoutFactory (sessionManager: SessionManager) {
+  // this function assumes it is invoked after a middleware that injects the user did in the request object
```

Comment:
Maybe we can validate this

Reply:
Definitely we can do it, but it will make the code too defensive. The logout should work as any protected endpoint because it is a protected endpoint, it assumes the user is logged in. The authentication check occurs in the middleware, not here, I think is not its responsibility